web/src/transmission.js

```diff
@@ -961,22 +999,26 @@ TODO: fix this when notifications get fixed
     const trackers = this._getTrackerCounts();
     const sitenames = Object.keys(trackers).sort();
 
-    // build the new html
-    let string = '';
-    string += this.filterTracker
-      ? '<option value="all">All</option>'
-      : '<option value="all" selected="selected">All</option>';
-    for (const sitename of sitenames) {
-      string += `<option value="${sitename}"`;
-      if (sitename === this.filterTracker) {
-        string += ' selected="selected"';
+    // Update select box only when list of trackers has changed
+    if (
+      sitenames.length !== this.oldTrackers.length ||
+      sitenames.some((x) => !this.oldTrackers.includes(x))
+    ) {
+      this.oldTrackers = [...sitenames];
```

Comment:
I don't think copy is needed here.

```suggestion
      this.oldTrackers = sitenames;
```

Reply:
Interesting, I thought the array it made a var of will be checked upon itself if without a deep copy, looks like I'm mistaken.